src/Layer/PointCloudLayer.js

```diff
@@ -0,0 +1,595 @@
+    }
+    return bbox;
+}
+
+export function parseMetadata(metadata, layer) {
+    layer.metadata = metadata;
+
+    var customBinFormat = true;
+
+    // Lopocs pointcloud server can expose the same file structure as PotreeConverter output.
+    // The only difference is the metadata root file (cloud.js vs infos/sources), and we can
+    // check for the existence of a `scale` field.
+    // (if `scale` is defined => we're fetching files from PotreeConverter)
+    if (layer.metadata.scale != undefined) {
+        layer.isFromPotreeConverter = true;
+        // PotreeConverter format
+        customBinFormat = layer.metadata.pointAttributes === 'CIN';
+        // do we have normal information
+        const normal = Array.isArray(layer.metadata.pointAttributes) &&
+            layer.metadata.pointAttributes.find(elem => elem.startsWith('NORMAL'));
+        if (normal) {
+            layer.material.defines[normal] = 1;
+        }
+    } else {
+        // Lopocs
+        layer.metadata.scale = 1;
+        layer.metadata.octreeDir = `itowns/${layer.table}.points`;
+        layer.metadata.hierarchyStepSize = 1000000; // ignore this with lopocs
+        customBinFormat = true;
+    }
+
+    layer.parse = customBinFormat ? PotreeCinParser.parse : PotreeBinParser.parse;
+    layer.extension = customBinFormat ? 'cin' : 'bin';
+    layer.supportsProgressiveDisplay = customBinFormat;
+}
+
+class PointCloudLayer extends GeometryLayer {
+    /**
+     * Constructs a new instance of point cloud layer.
+     * @constructor
+     * @extends GeometryLayer
+     *
+     * @example
+     * // Create a new point cloud layer
+     * const points = new PointCloudLayer('points',
+     *  {
+     *      url: 'https://pointsClouds/',
+     *      file: 'points.js',
+     *  }, view);
+     *
+     * View.prototype.addLayer.call(view, points);
+     *
+     * @param      {string}  id - The id of the layer, that should be unique. It is
+     * not mandatory, but an error will be emitted if this layer is added a
+     * {@link View} that already has a layer going by that id.
+     * @param      {object}  config   configuration, all elements in it
+     * will be merged as is in the layer.
+     * @param {string} config.url The base url.
+     * @param {string} config.file file name.
+     * @param {number} [config.pointBudget=2000000] max displayed points count.
+     * @param {number} [config.sseThreshold=2] screen space error Threshold.
+     * @param {Object} [config.fetchOptions={}] fetch config.
+     * @param {number} [config.pointSize=4] point size.
+     * @param {THREE.Material} [config.material] override material.
+     * @param {number} [config.mode=MODE.COLOR] displaying mode.
+     *
+     * @param  {View}  view  The view
+     */
+    constructor(id, config, view) {
+        super(id, new THREE.Group());
+        this.isPointCloudLayer = true;
+        this.protocol = 'potreeconverter';
+
+        this.file = config.file || 'cloud.js';
```

Comment:
Not sure if a default value should be set.

Reply:
i remove the default value.

Reply:
and added mandatory config properties